lib/formatOrder.js

```diff
@@ -0,0 +1,16 @@
+var sorting = require('postcss-sorting');
+
+function formatOrder(root, params) {
+  var sortOrder = params.stylelint['declaration-block-properties-order'];
+  if (!Array.isArray(sortOrder)) {
```

Comment:
I'd like to consider with at-rules e.g. `@apply`, `@include` of SCSS and `@mixin` of postcs-mixins. I think, at-rules in declaration nodes of PostCSS should be on the top of its declaration.

```
.selector {
  font-size: 12px;
  max-width: 280px;
  @include truncate();
}
```

The above code will be formatted to the following code.

```
.selector {
  @include truncate();
  max-width: 280px;
  font-size: 12px;
}
```

What do you think?